internal/repository/database/mysqldb/implementation.go

```diff
@@ -0,0 +1,166 @@
+package mysqldb
+
+import (
+	"context"
+	aulogging "github.com/StephanHCB/go-autumn-logging"
+	"time"
+
+	"github.com/eurofurence/reg-mail-service/internal/entity"
+	"github.com/eurofurence/reg-mail-service/internal/repository/config"
+	"github.com/eurofurence/reg-mail-service/internal/repository/database/dbrepo"
+	"github.com/eurofurence/reg-mail-service/internal/repository/logging"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+type MysqlRepository struct {
+	db *gorm.DB
+}
+
+func Create() dbrepo.Repository {
+	return &MysqlRepository{}
+}
+
+func (r *MysqlRepository) Open() error {
+	gormConfig := gorm.Config{}
```

Comment:
please set a table prefix (see latest attendee service PR), so it's possible to share the same mysql DB.